evap/staff/templates/staff_semester_preparation_reminder.html

```diff
@@ -10,6 +10,16 @@
 {% block content %}
     {{ block.super }}
 
+    {% csrf_token %}
```

Comment:
Hmm, I think I told you that you need to handle the csrf token here - guess I was wrong. Django has some doku entries about how to do this [here](https://docs.djangoproject.com/en/2.2/ref/csrf/#setting-the-token-on-the-ajax-request).
We already have some places where we do something similar. Our implementation is in `static/js/crsf_utils.js`. From a quick view, it looks like you don't need to do any additional handling, it should work automatically. Can you verify that?